deploy/examples/create-external-cluster-resources.py

```diff
@@ -791,9 +892,26 @@ def create_rgw_admin_ops_user(self):
                 err_msg = "failed to execute command %s. Output: %s. Code: %s. Error: %s" % (
                     cmd, execErr.output, execErr.returncode, execErr.stderr)
                 raise Exception(err_msg)
+        
 
+        # separately add info=read caps, because sometimes users already exited and the cap doesn't update
+        info_cap_supported = True
+        cmd = ['radosgw-admin', 'caps', 'add', '--uid', self.EXTERNAL_RGW_ADMIN_OPS_USER_NAME,
+            '--caps', 'info=read']
+        try:
+            output = subprocess.check_output(cmd,
+                                            stderr=subprocess.PIPE)
+        except subprocess.CalledProcessError as execErr:
+            # if the ceph version not supported for adding `info=read` cap(rgw_validation)
+            if execErr.stderr.decode("utf-8") == 'could not add caps: unable to add caps: info=read\n' and execErr.returncode == 244:
```

Comment:
can we do something like `strings.Contains` instead?

Reply:
Actually, it works perfectly as the canary test is running on pacific and I tested with it.